www/_posts/2017-12-20-plugins-browser-release.md

```diff
@@ -0,0 +1,101 @@
+---
+layout: post
+author:
+    name: Steve Gill
+    url: https://twitter.com/stevesgill
+title:  "Plugins & Cordova-Browser Release"
+categories: news
+tags: release plugins
+---
+
+The following plugins were updated today:
+
+* cordova-plugin-battery-status@2.0.0
+* cordova-plugin-camera@4.0.0
+* cordova-plugin-contacts@3.0.1
+* cordova-plugin-device@2.0.0
+* cordova-plugin-dialogs@2.0.0
+* cordova-plugin-file@6.0.0
+* cordova-plugin-geolocation@4.0.0
+* cordova-plugin-globalization@1.0.9
+* cordova-plugin-inappbrowser@2.0.0
+* cordova-plugin-media@5.0.0
+* cordova-plugin-media-capture@3.0.0
+* cordova-plugin-network-information@2.0.0
+* cordova-plugin-splashscreen@5.0.0
+* cordova-plugin-statusbar@2.4.0
+* cordova-plugin-screen-orientation@3.0.0
+* cordova-plugin-vibration@3.0.0
+
+`cordova-browser` was also updated today to `5.0.2`    
```

Comment:
Does it make sense to wait until cordova-browser 5.0.3 is out before announcing it's changes?

Reply:
yup. I'm going to remove browser from this blog post